src/coordinator/roles.spec.js

```diff
@@ -0,0 +1,121 @@
+import Roles from './roles';
+import * as objectUtils from '../utils/objects';
+
+describe('Coordinator.roles', function() {
```

Comment:
Nitpicky, but for the other submodules, we've used the `Coordinator/Roles` syntax